.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,52 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Version
+      description: Please specify all relevant version information (SwiftCurrent version, Xcode Version, Swift Version, etc.)
+      placeholder: |
+        SwiftCurrent Version: v4.5.17
+        Xcode Version: 13.2.1
+        Swift Verison: 5.5
```

In this example they seem like they might be using markdown lists because in our example it renders as one line. Could we add some dashes maybe?

```yaml
- type: textarea
  attributes:
    label: Steps To Reproduce
    description: Steps to reproduce the behavior.
    placeholder: |
      1. In this environment...
      2. With this config...
      3. Run '...'
      4. See error...
```
```suggestion
      placeholder: |
        - SwiftCurrent Version: v4.5.17
        - Xcode Version: 13.2.1
        - Swift Verison: 5.5
```

---

[Looks like that's right](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#example-1)

---

[This renders just fine](https://github.com/wwt/SwiftCurrent/blob/issue-forms/.github/ISSUE_TEMPLATE/bug_report.yml), do you want a list for another reason?